Packs/Arduino/Integrations/Arduino/Arduino.py

```diff
@@ -0,0 +1,162 @@
+import socket
+
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+
+''' CONSTANTS '''
+
+
+''' CLIENT CLASS '''
+
+
+class Server():
+    def __init__(self, host, port):
+        self.host = host
+        self.port = port
+        self.validate()
+
+    def validate(self):
+        is_ip = False
+        try:
+            socket.inet_aton(self.host)
+            is_ip = True
+        except socket.error:
+            is_ip = False
+        if not is_ip:
+            try:
+                self.host = socket.gethostbyname(self.host)
+            except Exception as err:
+                return_error(f"{self.host} is not a valid IP nor does not resolve to an IP - {err}")
+
+    def test_connect(self) -> str:
+        try:
+            with socket.socket(socket.AF_INET, socket.SOCK_STREAM) as s:
+                s.connect((self.host, self.port))
+                s.sendall(b"test")
+                data = s.recv(1024)
+                if data.decode().startswith("Response"):
+                    return 'ok'
+                else:
+                    return f"There was an issue. Check the Arduino code - {data}"
+        except Exception as err:
+            return err
+
+    def send_data(self, value) -> str:
+        try:
+            with socket.socket(socket.AF_INET, socket.SOCK_STREAM) as s:
+                s.connect((self.host, self.port))
+                s.sendall(value.encode())
+                data = s.recv(1024)
+                return data
+        except Exception as err:
+            return_error(err)
+
+
+''' HELPER FUNCTIONS '''
+
+
+''' COMMAND FUNCTIONS '''
+
+
+def test_module(server: Server) -> str:
+    return server.test_connect()
+
+
+def arduino_set_pin_command(server: Server, args: any) -> str:
+    pin_type: str = args.get('pin_type')
+    prefix: str = "Arduino.DigitalPins" if pin_type == "digital" else "Arduino.AnalogPins"
+    pin_number: int = int(args.get('pin_number'))
+    value: int = int(args.get('value'))
+    result: int = int(server.send_data(f"set:{pin_type}:{pin_number},{value}"))
```

Comment:
Please wrap those lines with `try-catch` for the same explanation I wrote on the main function and `return_error` in such cases.

Reply:
@aburt-demisto please check if you missed this comment